docs/features/pastefromlibreoffice/README.md

```diff
@@ -0,0 +1,85 @@
+---
+category: inserting-content
+order: 60
+url: features/pastefromlibreoffice
+menu-title: Paste from LibreOffice
+meta-title-short: Paste from LibreOffice
+---
+<!--
+Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md.
+-->
+
+# Pasting Content from LibreOffice
+
+<info-box info="">
+    This feature is provided through the <a href="https://ckeditor.com/cke4/addon/pastefromlibreoffice">Paste from LibreOffice</a> plugin that is included in the Standard and Full presets available from the official CKEditor <a href="https://ckeditor.com/ckeditor-4/download/">Download</a> site. You can also {@link guide/dev/plugins/README add it to your custom build} with <a href="https://ckeditor.com/cke4/builder">online builder</a>.
+</info-box>
+
+The [Paste from LibreOffice](https://ckeditor.com/cke4/addon/pastefromlibreoffice) plugin allows you to paste content from LibreOffice Writer and maintain original content structure and formatting.
+
+When enabled, it automatically detects LibreOffice Writer content and transforms its structure and formatting to clean HTML.
+
+## Supported Features
+
+Paste from LibreOffice retains the following formatting:
+
+* Text formatting
+    * Text and background colors
+    * Font family, style and size
+    * Basic formatting (bold, italic, underline)
+    * Font effects (strikethrough, superscript, subscript)
+    * Heading levels
+    * Text alignment
+* Lists
+    * Numbered and bulleted lists
+    * Multilevel lists
+    * Different numbering formats (Roman, decimal, alphanumeric)
+    * Custom start number (e.g. you can start the list from number 4)
+* Tables
+    * Borders and shading
+    * Cell size (width and height)
+    * Cell alignment
+* Images
+
+Paste from LibreOffice maintains most of Google Docs text formatting features with some exceptions related to advanced styling, customizations, or stylings that are not supported by HTML.
+
+## Sample
+
+The following sample content from a LibreOffice Writer document:
+
+{@img assets/img/pastefromlibreoffice_01.png 804 A sample LibreOffice Writer document.}
```

Comment:
This image seems very blurry. It would be nice to have some HIDPI images instead.

Reply:
These images are hi-dpi… This is how LO looks on my macbook 🤷‍♂ 
<img width="1073" alt="Comparison between Word and LO: Word view is sharp while LO's one – blurry" src="https://user-images.githubusercontent.com/1078728/74725069-a69cd480-523d-11ea-9483-2fa723cfad56.png">

Reply:
Seems like it will be fixed in 6.4.1: https://bugs.documentfoundation.org/show_bug.cgi?id=122218

Reply:
That's a bummer. It seems fine on Linux Fedora, but I don't have hidpi, so I can't help much. @AnnaTomanek could you take a look if we will be fine with such quality of the images? They don't look good IMO ;/